nidaqmx_examples/playrec.py

```diff
@@ -0,0 +1,83 @@
+"""Simultaneous read and write with NI USB-4431 or similar device."""
+
+import numpy as np
+import nidaqmx as ni
+from nidaqmx.constants import WAIT_INFINITELY
+
+
+def query_devices():
+  local = ni.system.System.local()
+  driver = local.driver_version
+
+  print(f'DAQmx {0}.{1}.{2}'.format(
+    driver.major_version, driver.minor_version, driver.update_version))
+
+  for device in local.devices:
+    print('Device Name: {0}, Product Category: {1}, Product Type: {2}'.format(
+      device.name, device.product_category, device.product_type))
+
+
+def playrec(
+  data, samplerate=96000, input_mapping=['Dev2/ai0', 'Dev2/ai1', 'Dev2/ai2'],
+  output_mapping=['Dev2/ao0']
```

Comment:
default parameters for these are too application-specific. can you remove them and pass them in __main__?

Reply:
Good idea.